docs/dev/RELEASE.md

```diff
@@ -70,12 +70,27 @@
 9. If something goes wrong, delete the tag and release in GitHub, fix the
    problem and try again.
 
-10. Update the Homebrew formula for Dafny (see below).
+10. Make a documentation snapshot
+   a. If you have not already, make a personal clone of the 
+      dafny-lang/dafny-lang.github.io repo
+   b. Be sure that repo has a clean status
+   c. In the dafny-lang/dafny repo, execute the script
+      `docs/make-snapshot X.Y.Z`, where X.Y.Z is the new release number
```

Comment:
I think this assumes that you have GNU bash, should we add instructions for running on Windows?

Reply:
Yes it is bash. Do we usually make all these scripts cross-platform? When it is in CI we can pick one platform to run on.

Reply:
I usually use Python and I think @MikaelMayer uses JS for better portability.  It doesn't matter as much if/when this runs in CI, I agree.